angular/src/lib/badge/tests/badge.component.spec.ts

```diff
@@ -1,75 +1,114 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component } from '@angular/core';
 import { BadgeComponent } from '../badge.component';
 
-describe('BadgeComponent', () => {
-  let testComponent: BadgeComponent;
-  let fixture: ComponentFixture<BadgeComponent>;
-  let badgeNativeElement: HTMLElement;
+@Component({
+  template: `
+    <cui-badge [ngClass]="'first second'" class="third fourth" color="blue" [rounded]="true">Testing Classes</cui-badge>
```

Comment:
Since this template has more than 2 attributes - can we add returns after each one?

Reply:
updated.